src/types/error.js

```diff
@@ -38,8 +38,9 @@ export class DustSpendError extends Error {
 export class InsufficientFundsError extends Error {
   name: string
   +currencyCode: string | void
+  +amount: string | void
```

Comment:
I'm not sure what this means. Is it `sentAmount`, or `availableAmount`? Maybe add a comment, or improve the name.